packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.tsx

```diff
@@ -87,37 +81,52 @@ export class PersonaCoin extends React.Component<IPersonaProps, IPersonaState> {
     const {
       coinProps,
       coinSize,
+      getStyles: getStylesProp,
       imageUrl,
       onRenderCoin = this._onRenderCoin,
       onRenderInitials = this._onRenderInitials,
+      presence,
+      theme,
     } = this.props;
 
     const size = this.props.size as PersonaSize;
     const divProps = getNativeProps(this.props, divProperties);
     const coinSizeStyle = coinSize ? { width: coinSize, height: coinSize } : undefined;
 
+    const personaPresenceProps: IPersonaPresenceProps = {
+      coinSize,
+      presence,
+      theme,
+      size,
+    };
+
+    const classNames = getClassNames(getStylesProp! || getStyles!, {
```

Comment:
Do we have to use bangs here and elsewhere? I.e. are we absolutely certain getStylesProp/getStyles/theme can never be null? If they can be null, we should avoid using !, to allow typescript to do its nullable type checking

Reply:
I know theme can't be null. It needs to pull the default theme values. I'll test removing the bang from getStyles. I'm sure I put it there in the middle of converting as it was yelling about types, but I probably fixed that by the end.